ksqldb-functional-tests/src/test/resources/sql-tests/test.sql

```diff
@@ -57,7 +57,7 @@ ASSERT VALUES bar (rowtime, id, col1) VALUES (1, 1, 1);
 --@test: basic test with key_schema_id (format AVRO)
 ----------------------------------------------------------------------------------------------------
 CREATE STREAM foo (id INT KEY, col1 INT) WITH (kafka_topic='foo', key_format='AVRO', value_format='AVRO', key_schema_id=123);
-ASSERT STREAM foo (id INT KEY, col1 INT) WITH (kafka_topic='foo', key_format='AVRO', value_format='AVRO', key_schema_id=123);
```

Comment:
Why is this being removed? The test is called `basic test with key_schema_id`.

Reply:
It was added by my previos PR when putting schema id to format properties: https://github.com/confluentinc/ksql/commit/84e176546b759bcf32f242dd959dfc2a48ce685c. Not we don't put it there, it's deleted

Reply:
Then we should delete the entire test, right? The test currently no longer tests what the name of the test suggests.